packages/cli/src/builders/repo-builder.ts

```diff
@@ -75,32 +85,78 @@ export function createRepo(options: Options) {
     undefined,
     propsFactories
   );
-
   const repoPosition = repoClassDec.getChildIndex();
 
-  sourceFile.insertVariableStatement(repoPosition, {
-    declarationKind: VariableDeclarationKind.Const,
-    isExported: isFunctionTpl,
-    declarations: [
-      {
-        name: resolveStoreVariableName(options.template, storeNames),
-        initializer: `new Store({ name: '${storeNames.propertyName}', state, config })`,
-      },
-    ],
-  });
+  if (isStoreInsideClass && repoClassDecConstructor) {
+    const lastPropertyIndex = repoClassDec
+      .getLastChildByKind(SyntaxKind.PropertyDeclaration)
```

Comment:
Can we make it cleaner and abstract this chunk to a different place?

Reply:
Are you about the whole `if` block?

Reply:
Yes